mlir/lib/Conversion/ArithAndMathToAPFloat/CMakeLists.txt

```
@@ -0,0 +1,50 @@
+add_mlir_library(ArithAndMathToAPFloatUtils
```

Shouldn't this be?
```suggestion
add_mlir_library(MLIRArithAndMathToAPFloatUtils
```
This is the only MLIR library without the MLIR prefix.

---

I can fix that in the next one

---

It broke our downstream CI :)